public/app/features/dashboard-scene/serialization/ElementPanelMappingService.ts

```diff
@@ -0,0 +1,60 @@
+// function to manage a lookup table that will hold element_identifer : panel_id
+
+// function to manage a lookup table that will hold element_identifer : panel_id
+export interface ElementPanelMapping {
+  [elementIdentifier: string]: number;
+}
+
+export class ElementPanelMappingService {
```

Comment:
I'm not convinced we need this abstraction, it's literally  a `Map`.

Reply:
good catch, remove the singleton, made a Map